timeseries/setup.py

```diff
@@ -29,8 +29,8 @@
     "pandas",  # version range defined in `core/_setup_utils.py`
     "statsmodels>=0.13.0,<0.14",
     "gluonts>=0.12.4,<0.13",
-    "torch>=1.9,<1.14",
-    "pytorch-lightning>=1.7.4,<1.10.0",
+    "torch>=1.9,<2.1",
```

Comment:
Due to security vulnerabilities of torch, we may need to force "torch>=1.13.1"

Reply:
in that case, the minimum torchvision version would go to 0.14.0 and your 0.12 comment becomes obsolete 👍

Reply:
Do we want to be supporting multiple major versions of torch? That sounds intimidating to maintain. Is it possible to drop support of torch 1.x and only support 2.x?

Reply:
e.g. torch-lightning supports 1.11-2.0, I think it should be ok to support v1 a while longer:

> torch.compile is the main API for PyTorch 2.0, which wraps your model and returns a compiled model. It is a fully additive (and optional) feature and hence 2.0 is 100% backward compatible by definition.

ref https://pytorch.org/blog/pytorch-2.0-release/

personally I as user would be fine with >=2 as well

Reply:
because of `pytorch-lightning>=2.0.0,<2.1` (https://github.com/autogluon/autogluon/pull/3190/commits/6a0387e6ef8cfe838e27093831061909dd7b3985), shall I consequently bump

```diff
- torch>=1.9,<2.1
+ torch>=1.11,<2.1
- torchvision>=0.10.0,<0.16
+ torchvision>=0.12.0,<0.16 
```
?